src/Evolution/Systems/ScalarAdvection/Tags.hpp

```diff
@@ -0,0 +1,34 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <string>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/DataVector.hpp"  // IWYU pragma: keep
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+namespace ScalarAdvection {
+/*!
+ * \brief Tags for the ScalarAdvection system
+ */
+namespace Tags {
+/// The scalar field to evolve
+struct U : db::SimpleTag {
+  using type = Scalar<DataVector>;
```

Comment:
Another question for @nilsdeppe: Do you have an opinion on whether or not we should template these system tags on `DataType` and `Frame`? We haven't done that super consistently so far. We should probably make a decision and then be more consistent in the future.

I'm bringing this up now because it would be annoying to refactor this new system later when we decide the tags should have the template parameters.

Reply:
Other than the GH/GR equations, I don't know why we would need to use them with a different `DataType`. Changing the frame seems especially dangerous since by definition the `Inertial` frame is the frame where the PDEs are written in their "standard" form.